.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### Summary
+
+(Guidelines for creating a bug report are available
+here: https://github.com/capistrano/capistrano/blob/master/DEVELOPMENT.md)
+
+Provide a general description of the code changes in your pull
+request... were there any bugs you had fixed? If so, mention them. If
+these bugs have open GitHub issues, be sure to tag them here as well,
+to keep the conversation linked together.
+
+### Short checklist
+
+[ ] Did you run `bundle exec rubocop -a` to fix linter issues?
+[ ] If relevant, did you create a test?
+[ ] Did you confirm that the RSpec tests pass?
```

Comment:
To get these to appear as actual checkboxes in GitHub, I believe the correct Markdown syntax is:

```
- [ ] 
```